packages/es-dev-server/src/utils/utils.js

```diff
@@ -114,6 +114,22 @@ export class SSEStream extends Stream.Transform {
   }
 }
 
+/**
+ * @param  {*}  x
+ * @return {x is boolean}
+ */
+export function isBoolean(x) {
```

Comment:
Not sure if it's worth creating a helper function for this? `typeof x === 'boolean'` is fairly straightforward?

Reply:
except typescript makes the straightforward crooked.

the return type `x is boolean` is a necessary typeguard to let options be passed as `WatchOptions` and not `boolean|WatchOptions`

Reply:
ok, makes sense.